app/profile-page.html

```diff
@@ -13,19 +13,25 @@
       <!-- endbuild -->
       <!-- build:css styles/main.css -->
       <link rel="stylesheet" href="styles/main.css">
+	  <link rel="stylesheet" href="styles/partials/profile.css">
       <!-- endbuild -->
```

Comment:
You should just be able to import this in main.scss using `@import 'partials/_profile';` at the bottom of the file- Make sure you add the underscore